src/main/resources/MainScreen.fxml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is this?

Reply:
idk it just there

Reply:
sorry I should have been more clear. i mean this entire file.